features/rt/manifests/06-mc-rtkernel-worker-rt.yaml

```diff
@@ -20,7 +20,7 @@ spec:
+                      source: data:text/plain;charset=utf-8;base64,IyEvYmluL2Jhc2gKCnNldCAtZXVvIHBpcGVmYWlsCgpSRVBPX0RJUj0iL2V0Yy95dW0ucmVwb3MuZCIKUlRfUkVQTz0iJHtSRVBPX0RJUn0vcnQta2VybmVsLnJlcG8iCgojIEVuYWJsZSB5dW0gcmVwbwppZiBbWyAtZiAkUlRfUkVQTyBdXQp0aGVuCiAgIyBUaGUgZW52IHZhciBtaWdodCBoYXZlIGJlZW4gY2hhbmdlZCwgc28gYWx3YXlzIGNyZWF0ZSBuZXcgcnQgcmVwbwogIHJtICRSVF9SRVBPCmZpCgpta2RpciAtcCAkUkVQT19ESVIKY2F0ID4gJFJUX1JFUE8gPDxFT0YKW3J0XQpiYXNldXJsPSR7UlRfUkVQT19VUkx9CmdwZ2NoZWNrPTAKRU9GCgojIHVwZGF0ZSBjYWNoZQpycG0tb3N0cmVlIHJlZnJlc2gtbWQgLWYKCiMgSW5zdGFsbCBwYXRjaGVkIG1pY3JvY29kZQojIHNlZSBodHRwczovL3NyYy5vc2NpLnJlZGhhdC5jb20vcnBtcy9taWNyb2NvZGVfY3RsL3B1bGwtcmVxdWVzdC85CnJlcGxhY2VkUGFja2FnZXM9JChycG0tb3N0cmVlIHN0YXR1cyAtLWpzb24gfCBqcSAnLmRlcGxveW1lbnRzW10gfCBzZWxlY3QoLmJvb3RlZCA9PSB0cnVlKSB8IC4ibGF5ZXJlZC1jb21taXQtbWV0YSIuInJwbW9zdHJlZS5yZXBsYWNlZC1iYXNlLXBhY2thZ2VzIicpCmlmIFtbICRyZXBsYWNlZFBhY2thZ2VzID1+ICJtaWNyb2NvZGVfY3RsIiBdXQp0aGVuCiAgICBlY2hvICJtaWNyb2NvZGVfY3RsIHBhdGNoIGFscmVhZHkgaW5zdGFsbGVkIgplbHNlCiAgICBlY2hvICJJbnN0YWxsaW5nIG1pY3JvY29kZV9jdGwgcGF0Y2giCiAgICBycG0tb3N0cmVlIG92ZXJyaWRlIHJlcGxhY2UgJHtNSUNST0NPREVfVVJMfQpmaQoKIyBTd2FwIHRvIFJUIGtlcm5lbAprZXJuZWw9JCh1bmFtZSAtYSkKaWYgW1sgJGtlcm5lbCA9fiAiUFJFRU1QVCBSVCIgXV0KdGhlbgogICAgZWNobyAiUlQga2VybmVsIGFscmVhZHkgaW5zdGFsbGVkLCBjaGVja2luZyBmb3IgdXBkYXRlcyIKICAgIHJwbS1vc3RyZWUgdXBncmFkZSAtLXVuY2hhbmdlZC1leGl0LTc3CiAgICBpZiBbWyAkPyAtZXEgNzcgXV0KICAgIHRoZW4KICAgICAgZWNobyAiTm8gdXBkYXRlIGF2YWlsYWJsZSwgbm90aGluZyB0byBkbyIKICAgIGVsc2UKICAgICAgZWNobyAiUlQga2VybmVsIHVwZGF0ZWQsIHJlYm9vdGluZyIKICAgICAgc3lzdGVtY3RsIHJlYm9vdAogICAgZmkKZWxzZQogICAgZWNobyAiSW5zdGFsbGluZyBSVCBrZXJuZWwiCiAgICBycG0tb3N0cmVlIG92ZXJyaWRlIHJlbW92ZSBrZXJuZWx7LC1jb3JlLC1tb2R1bGVzLC1tb2R1bGVzLWV4dHJhfSAtLWluc3RhbGwga2VybmVsLXJ0LWNvcmUgLS1pbnN0YWxsIGtlcm5lbC1ydC1tb2R1bGVzIC0taW5zdGFsbCBrZXJuZWwtcnQtbW9kdWxlcy1leHRyYQogICAgZWNobyAiUmVib290aW5nIgogICAgc3lzdGVtY3RsIHJlYm9vdApmaQo=
```

Comment:
@slintes @MarSik how can we make patch reviews possible with these blobs?

Reply:
with #36 the generated manifests are not committed anymore but created on the fly during deployment. See https://github.com/openshift-kni/baremetal-deploy/pull/36#pullrequestreview-329700002